test/jobs/benchmark_regression_job_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+
+class BenchmarkRegressionJobTest < ActiveJob::TestCase
+
+  test "No Benchmark Regression Detected" do
+    benchmark_run, initiator, benchmark_type, benchmark_result_type = setup_benchmarks
```

Comment:
You can call this in a [setup method](https://github.com/seattlerb/minitest#unit-tests)